server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoErrorRetryAnalyzer.ts

```diff
@@ -51,11 +62,7 @@ export class MongoErrorRetryAnalyzer {
             return this.getRetryRuleFromSet(error, this.mongoNetworkErrorRetryRuleset);
         }
 
-        if (error.name === "MongoError") {
-            return this.getRetryRuleFromSet(error, this.mongoErrorRetryRuleset);
-        }
-
-        return this.defaultRule;
+        return this.getRetryRuleFromSet(error, this.mongoErrorRetryRuleset);
```

Comment:
The intention here is for this to be a general catch-all retry rule, correct?

Reply:
From what I know, some mongo errors don't have the name property, like the bulkwrite timeout request (some does though). And this line would catch these errors and do retry. Xin, please correct me if I miss anything

Reply:
@arikt-ms , yes, @yunho-microsoft  said right. Originally based on error we have, sdk always have the error name property. However, from most recent error some of these don't have these property therefore we didn't retry